src/main/java/com/jcabi/xml/ClasspathSources.java

```diff
@@ -81,17 +85,38 @@ public ClasspathSources(final String pfx) {
     @Override
     public Source resolve(final String href, final String base)
         throws TransformerException {
-        final InputStream stream = this.getClass().getResourceAsStream(
-            String.format("%s%s", this.prefix, href)
+        InputStream stream = this.getClass().getResourceAsStream(
+            String.format(PATTERN, this.prefix, href)
         );
         if (stream == null) {
-            throw new TransformerException(
-                String.format(
-                    "resource \"%s\" not found in classpath with prefix \"%s\"",
-                    href, this.prefix
-                )
+            stream = this.getClass().getResourceAsStream(
+                String.format(PATTERN, base, href)
             );
+            if (stream == null) {
+                throw this.getTransformerException(href, base);
+            }
         }
         return new StreamSource(stream);
     }
+
+    /**
+     * Create TransformerException with message.
+     * @param href Resource identifier
+     * @param base Resource base
+     * @return New TransformerException with message
+     */
+    private TransformerException getTransformerException(final String href,
```

Comment:
@piddubnyi instead of adding this method, let's keep the `String.format` we had before and just use the new `base` parameter - no need for a `StringBuilder`